crits/samples/api.py

```diff
@@ -83,6 +84,7 @@ def obj_create(self, bundle, **kwargs):
         related_type = bundle.data.get('related_type', None)
         backdoor_name = bundle.data.get('backdoor_name', None)
         backdoor_version = bundle.data.get('backdoor_version', None)
+        description = bundle.data.get('description', None)
```

Comment:
It's moved from the past commit, not copied. Unless I messed up somehow. Before it was under metadata, but it should be able to assign a descrip regardless.